graphics/libjpeg/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        thorfdbg libjpeg c719010a26ce0c666e98b2acf924ad5fc24b4f5d
+# see https://github.com/thorfdbg/libjpeg/blob/master/README
+version             1.70
+revision            0
+categories          graphics
+license             GPL-3
+maintainers         nomaintainer
+
+description         Complete implementation of 10918-1 (JPEG) from jpeg.org
+
+long_description    A complete implementation of 10918-1 (JPEG) coming \
+                    from jpeg.org (the ISO group) with extensions for \
+                    HDR standardized as 18477 (JPEG XT).
+
+checksums           rmd160  245dc39bc712770499a86d904e8aa332a2118c1a \
+                    sha256  62e8c8b1898a7e9195c90b673e6db0090b474372b4e7f485e471f37a08852e18 \
+                    size    560772
+
+distname            ${name}-${version}
+
+use_autoreconf      yes
+
+build.target        doc final \
+                    SETTINGS=clang
+
+destroot.post_args-append \
+                    INSTALLDIR=${destroot}${prefix}/bin \
+                    SETTINGS=clang
```

Build systems are normally told how to install into the destroot by the `DESTDIR` variable which MacPorts sets for you in `destroot.destdir`. For build systems like this one that don't honor that, it's customary to override `destroot.destdir` with the required value.

You can avoid repeating the `SETTINGS` variable by reusing the one already set in `build.args`.

```suggestion
destroot.destdir    INSTALLDIR=${destroot}${prefix}/bin
destroot.args       {*}${build.args}
```

---

Thanks, done, except there's no such variable `build.args`.